scripts/demo.sh

```
@@ -1,128 +1,408 @@
-  "product_service_type": {
-    "type": "STANDARD_CURRENT_ACCOUNT"
-  },
-  "account_currency": "GBP"
-}' localhost:9091 meridian.current_account.v1.CurrentAccountService/InitiateCurrentAccount)
-
-ACCOUNT_ID=$(echo "$CREATE_RESPONSE" | jq -r '.currentAccountFacilityReference')
-echo -e "${GREEN}✓ Account Created:${NC} $ACCOUNT_ID\n"
-
-# Step 2: Execute Deposit
-echo -e "${BLUE}════ Step 2: Execute Deposit (£100) ════${NC}"
+  "account_identification": "ACC-DEMO-'$(date +%s)'",
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)